cmake/modules/LXQtTranslateTs.cmake

```diff
@@ -63,7 +63,8 @@
 #
 
 # CMake v2.8.3 needed to use the CMakeParseArguments module
-cmake_minimum_required(VERSION 2.8.3 FATAL_ERROR)
+# We check against 3.6.2 before so this check could go away imho
+cmake_minimum_required(VERSION 3.6.2 FATAL_ERROR)
 
```

Comment:
The check is not needed here imho - or is there a special reason to check again?

Reply:
erm - it's a test/set and it is needed, i also put it into the compiler settings